go.mod

```diff
@@ -0,0 +1,3 @@
+module main
```

Comment:
should you be adding this?

Reply:
For now, yes. This is required to get "replace" instructions to work in other projects that depend on this.